cmd/cli/logs/logs.go

```diff
@@ -49,10 +49,5 @@ func NewCmd() *cobra.Command {
 		`Follow the logs in real-time after retrieving the current logs.`,
 	)
 
-	logsCmd.PersistentFlags().BoolVarP(
-		&options.Follow, "history", "h", false,
-		`Show all logs with history. TODO better definition`,
-	)
-
```

Comment:
Should the `WithHistory` option be present/doing anything?

Reply:
The intent was that by default we show everything so far, and then optionally follow. This was intended to allow you to just follow from the current output but I think we changed our mind on it quite late.

I've got a couple of other cleanups to do in logging so I'll do them in separate PRs.